src/MutableVertexPartition.cpp

```diff
@@ -306,6 +306,48 @@ vector<size_t> MutableVertexPartition::renumber_communities(vector<MutableVertex
  Renumber the communities using the original fixed membership vector. Notice
  that this doesn't ensure any property of the community numbers.
 *****************************************************************************/
+void MutableVertexPartition::renumber_communities(map<size_t, size_t> const& membership)
```

Comment:
I would follow the same pattern as for `vector<size_t> MutableVertexPartition::renumber_communities(vector<MutableVertexPartition*> partitions)`. Here we don't need the multiple partitions, so we can simpy use `vector<size_t> MutableVertexPartition::renumber_communities(map<size_t, size_t> const& membership)`, which should then return the new `membership` vector. In the line 348, we can then call `set_membership` with this new membership.